src/harness/override-prod.yaml

```diff
@@ -645,7 +660,7 @@ platform:
   scm-service:
     autoscaling:
       enabled: false
-    replicaCount: 1
+    replicaCount: 2
```

Comment:
With auto-scaling enabled, will the replica count go below 2 or act as the minReplica count?

Reply:
if autoscaling is enabled then customer will set minReplicas accordingly